src/sdk/webhook-consumer/WebhookConsumer.ts

```
@@ -249,7 +308,14 @@ export class WebhookConsumer {
 				const queueLength = await this.redisClient.lLen(this.queueName);
 				if (queueLength > 0) {
 					LogEngine.info(`Found ${queueLength} events in queue ${this.queueName}`);
+				} else {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(brace-style)